features/experimental-ui-theming.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
```

Comment:
You can just delete this file now https://github.com/duckduckgo/privacy-configuration/pull/3207 will merge. The code checks for another platform adding that flag and makes it a requirement only when it's on more than one.